src/components/calcite-card/calcite-card.scss

```diff
@@ -0,0 +1,119 @@
+:host {
+  max-width: 100%;
+  & > a {
+    display: flex;
+    height: 100%;
+    flex-direction: column;
+    justify-content: space-between;
+    background-color: var(--calcite-ui-foreground);
+    transition: $transition;
+    position: relative;
+    border: 1px solid var(--calcite-ui-border-2);
+    color: var(--calcite-ui-text-3);
+    box-shadow: 0 0 0 rgba(0, 0, 0, 0);
+    &:hover {
+      box-shadow: $shadow-1--hover;
+      z-index: 1;
+    }
+    &:active {
+      box-shadow: $shadow-1--press;
+      z-index: 1;
+    }
+  }
+}
+
+// todo
```

Comment:
should we get rid of these todo?

Reply:
well, I still need to finish them ;) But yeah, will work through and remove :)